src/polar/Pie.tsx

```diff
@@ -497,7 +497,7 @@ export class Pie extends PureComponent<Props, State> {
           tabIndex={-1}
           className="recharts-pie-sector"
           {...adaptEventsOfChild(this.props, entry, i)}
-          key={`sector-${entry?.startAngle}=${entry?.endAngle}`}
+          key={`sector-${entry?.startAngle}-${entry?.endAngle}-${entry.midAngle}`}
```

Comment:
does the guarantee uniqueness for each sector?

Reply:
👋 
- it looks like `midAngle` is the most and/or _only_ easily-identifiable unique field of each `entry` object (_each "slice" of the pie_)
- no duplicate-key lint err is present in any of the tests, including the new test, duplicate of the sandbox example which _did_ reproduce the key error in the test prior to adding this `midAngle` data piece to the key

Reply:
🚀 - good enough for a mitigation for now, lets monitor issues and see if this is brought up again after I merge and patch. Thank you for jumping in!

Reply:
Hi! Sorry for excavating this issue, but it is still present.
I've been having the warning below (logged as an error) when I have 2 or more cells with value equal to '0'.
`Warning: Encountered two children with the same key, 'sector-360-360-360'.`

Reply:
I can't reproduce https://codesandbox.io/p/sandbox/two-simple-pie-chart-forked-25xllj

You probably need to update recharts @VictorHugoLeme